.github/workflows/images.yaml

```diff
@@ -58,15 +79,32 @@ jobs:
 
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v3
-
+      
+      - name: Az CLI login
+        uses: azure/login@v1
+        if: ${{ github.event_name == 'merge_group' }}
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+        
       - name: Build Images
         shell: bash
         run: |
           set -euo pipefail
           echo "TAG=$(make version)" >> $GITHUB_ENV
-          make retina-image-win \
-            IMAGE_NAMESPACE=${{ github.repository }} \
-            PLATFORM=${{ matrix.platform }}/${{ matrix.arch }}
+          if [ "$IS_MERGE_GROUP" == "true" ]; then
+            az acr login -n acnpublic
+            make retina-image-win \
+              IMAGE_NAMESPACE=microsoft/retina \
```

Comment:
can be the same as {{ github.repository }}

Reply:
updated.